geomesa-core/src/test/scala/org/locationtech/geomesa/core/data/AccumuloDataStoreDefaults.scala

```diff
@@ -0,0 +1,109 @@
+package org.locationtech.geomesa.core.data
+
+import java.text.SimpleDateFormat
+import java.util.{Date, TimeZone}
+
+import org.geotools.data.{DataStoreFinder, DataStore}
+import org.geotools.factory.{CommonFactoryFinder, Hints}
+import org.geotools.feature.DefaultFeatureCollection
+import org.geotools.feature.simple.SimpleFeatureBuilder
+import org.geotools.geometry.jts.JTSFactoryFinder
+import org.geotools.referencing.crs.DefaultGeographicCRS
+import org.locationtech.geomesa.core.index._
+import org.locationtech.geomesa.feature.AvroSimpleFeatureFactory
+import org.locationtech.geomesa.utils.geotools.SimpleFeatureTypes
+import org.locationtech.geomesa.utils.text.WKTUtils
+import org.opengis.feature.simple.SimpleFeatureType
+
+import scala.collection.JavaConversions._
+
+/**
+ * Provides values and functions used for testing
+ * in AccumuloDataStoreTest and AccumuloFeatureStoreTest.
+ */
+trait AccumuloDataStoreDefaults {
+  val ff = CommonFactoryFinder.getFilterFactory2
+  val geotimeAttributes = org.locationtech.geomesa.core.index.spec
+  val hints = new Hints(Hints.FEATURE_FACTORY, classOf[AvroSimpleFeatureFactory])
+  val featureFactory = CommonFactoryFinder.getFeatureFactory(hints)
+  val WGS84 = DefaultGeographicCRS.WGS84
+  val gf = JTSFactoryFinder.getGeometryFactory
+
+  val df = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss")
+  df.setTimeZone(TimeZone.getTimeZone("UTC"))
+
+  val defaultSchema = "name:String,geom:Point:srid=4326,dtg:Date"
+  val defaultGeom = WKTUtils.read("POINT(45.0 49.0)")
+  val defaultDtg = new Date(100000)
+  val defaultName = "testType"
+  val defaultFid = "fid-1"
+
+  val defaultTable = "AccumuloDataStoreTest"
```

Comment:
you should make this something distinct like 'getClass.getSimpleName' or something, otherwise different unit tests will use the same mock table and conflicts may arise.